src/service/api.tsx

```diff
@@ -50,7 +50,7 @@ const api = {
     this.setStore(store);
   },
 
-  isIdInFavorites(id: number) {
+  isIdInFavorites(id: number | null) {
```

Comment:
I believe we dont need to do any checks in case of null
Honestly it should never be null by design unless there is a strong reason (like api from time to time returns is as null)

Reply:
same for other checks in service